src/components/v2/Layout/Footer/index.tsx

```diff
@@ -37,7 +38,12 @@ export const Footer: React.FC<IFooterProps> = ({ currentBlockNumber }) => {
       </Typography>
 
       <div css={styles.links}>
-        <a css={styles.link} href={ETHERSCAN_XVS_URL} target="_blank" rel="noreferrer">
+        <a
```

Comment:
Should we use typography here?

Reply:
We shouldn't no, the anchor tag is used as a wrapper only here, we don't display any text inside it.